Update active-directory-cloud-sync-how-to-install.md (#108855)
Azure/azure-content: includes/active-directory-cloud-sync-how-to-install.md

 >[!NOTE]
 >If you are installing the provisioning agent for use with [on-premsise app provisioning](../articles/active-directory/app-provisioning/on-premises-application-provisioning-architecture.md) then select **On-premises application provisioning (Azure AD to application)**.

 10. Sign in with your Azure AD global administrator or Hybrid Identity Administrator account.  If you have Internet Explorer enhanced security enabled, it will block the sign-in.  If so, close the installation, [disable Internet Explorer enhanced security](/troubleshoot/developer/browsers/security-privacy/enhanced-security-configuration-faq), and restart the **Azure AD Connect Provisioning Agent Package**  installation.

 :::image type="content" source="media/active-directory-cloud-sync-how-to-install/azure-ad-cloud-sync-sign-in-to-azure.png" alt-text="Screenshot of the Connect Azure AD screen."  lightbox="media/active-directory-cloud-sync-how-to-install/azure-ad-cloud-sync-sign-in-to-azure.png":::

>  11. On the **Configure Service Account** screen, select a group Managed Service Account (gMSA). This account is used to run the agent service. If a managed service account is already configured in your domain by another agent and you are installing a second agent, you should select **Create gMSA** as the system will detect the existing account and add the required permissions for the new agent to use the gMSA account. When prompted, choose either:

Reviewer: ```suggestion
 11. On the **Configure Service Account** screen, select a group Managed Service Account (gMSA). This account is used to run the agent service. If a managed service account is already configured in your domain by another agent and you are installing a second agent, select **Create gMSA** because the system will detect the existing account and add the required permissions for the new agent to use the gMSA account. When prompted, choose either:
```
Editorial fixes for Microsoft style

>  13. Sign in with your Active Directory domain administrator account.  The domain administrator account shouldn't have an expired password. In case the password is expired or changes during agent install, you'll need to reconfigure the agent with the new credentials. This operation will add your on-premises directory. Select **OK**, then select **Next** to continue. 

Reviewer: ```suggestion
 13. Sign in with your Active Directory domain administrator account.  The domain administrator account shouldn't have an expired password. In case the password is expired or changes during the agent installation, you'll need to reconfigure the agent with the new credentials. This operation will add your on-premises directory. Select **OK**, then select **Next** to continue. 
```
